File: 物件導向程式語言 C# 7答案檔/309.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSA03
{
    class CSA03
    {
        static void Main(string[] args)
        {
            try
            {
                int number = Convert.ToInt32(Console.ReadLine());
                if (number < 1 || number > 20) { throw new Exception(); }
                long x = compute(number);
                
                Console.Write(x.ToString());
            }
            catch
            {
                Console.Write("error");
            }
            Console.ReadKey();
        }
        // ==================================================
        // Please write your code in the specified Method
        // Do NOT change the method name, type of parameter
        // ==================================================
        static long compute(long num)
        {
            if (num == 1 )
                return 1;
            return compute(num - 1) * num;
        }
  
    }
}

File: 程式語言 V2答案檔/C/309.c
#include <stdio.h>
#include <stdlib.h>

double compute(int a[]);
int main (){
	double ans;
	int i,a[6];
	for(i = 0;i < 6; i++) {
        scanf("%d", &a[i]);
	}
	ans = compute(a);
	printf("%.3lf\n", ans);

	return 0;
}

// ===================================================
// Please write your code in the specified function
// Do NOT change the function name, type of parameter
// ===================================================
double compute(int a[])
{
	double min = 10.0;
	int i;
	for(i = 0;i < 3;i++){
    	if((double)a[i] / a[i+3] < min){
    		min =  (double)a[i] / a[i+3];
       }
	}
	return min;
}

